### ISSUE TITLE
Negative soil evaporation output

### ISSUE THREAD

**Issue by dschlaep** [DrylandEcology/rSOILWAT2#58]
```
slot(x, "EVAPSOIL")
Slot "Year":
      Year     Lyr_1          Lyr_2
 [1,] 1980  9.844250  -9.500000e+00
 [2,] 1981  8.497285  -1.000000e+01
 [3,] 1982  9.120529  -1.100000e+01
 [4,] 1983 14.194938  -1.600000e+01
 [5,] 1984 10.287924  -2.550000e+01
```
See https://github.com/Burke-Lauenroth-Lab/rSFSW2/issues/189 where this bug was first identified.

**Comment by dschlaep** [comment]
Apparently, `rSFSW2` was sending incorrect evaporation coefficients, e.g., `1 0 1 0 0 0` (fixed with https://github.com/Burke-Lauenroth-Lab/rSFSW2/pull/195).
=> why did `rSOILWAT2` accept such values
=> why did it or `SOILWAT2` produce negative evaporation given incorrect coefficients?
=> doesn't `SOILWAT2` re-scale those coefficients such that their sum == 1?

**Comment by dschlaep** [comment]
-> `rSOILWAT2` no longer accepts invalid values (addressed by 8aa3f1f6224ff3dba326cfb0ed4865b7c5c3c67d)
-> `SOILWAT2` considers the first layer with 0 or a negative value to mark the end of water extraction potential by evaporation or transpiration (see functions `nlayers_bsevap()` and `nlayers_vegroots()` https://github.com/DrylandEcology/SOILWAT2/commit/7272805a75a0653862410537a657b5328439669b)
-> `SOILWAT2` doesn't currently reject negative coefficients (see https://github.com/DrylandEcology/SOILWAT2/issues/287)
